eth-bytecode-db/eth-bytecode-db/README.md

```diff
@@ -0,0 +1,47 @@
+# Ethereum bytecode database
+
+
+## Development
+
+> all commands should be executed in `eth-bytecode-db/eth-bytecode-db` dir
+
+### Postgres
+
+```bash
+docker run -p 5432:5432 --name eth-bytecode-postgres -e POSTGRES_PASSWORD=admin -d postgres
+export DATABASE_URL=postgres://postgres:admin@localhost:5432/eth-bytecode-db
+```
+
+### Migrations
+
+1. Install `sea-orm-cli`:
+
+```bash
+cargo install sea-orm-cli
```

Comment:
Another option is to build `migration` crate and run migrations from that trait as `target/debug(release)/migration up (down, fresh)`. In that case it is not required to run migrations from `eth-bytecode-db/eth-bytecode-db` directory